templates/cassiopeia/scss/blocks/_footer.scss

```
@@ -22,14 +22,36 @@
     color: var(--cassiopeia-color-primary);
     background: $white;
     border-radius: $border-radius;
-  }
+    }
```

```suggestion
  }
```
Why are you doing unrelated and wrong code style changes?